clients/src/test/java/org/apache/kafka/common/network/SslSelectorTest.java

```diff
@@ -125,10 +125,17 @@ public void testConnectionWithCustomKeyManager() throws Exception {
         selector.send(createSend(node, request));
 
         waitForBytesBuffered(selector, node);
+        assertEquals(1, metrics.metrics().keySet().stream().
```

Comment:
We could have the same test in the non-SSL case which checks that the value is zero when there are connections.

Reply:
added